llarp/ev/ev_libuv.cpp

```diff
@@ -376,8 +36,8 @@ namespace libuv
       llarp::LogTrace("ticker_glue::OnTick() start");
       ticker_glue* ticker = static_cast<ticker_glue*>(t->data);
       ticker->func();
-      Loop* loop = static_cast<Loop*>(t->loop->data);
-      loop->FlushLogic();
+      // Loop* loop = static_cast<Loop*>(t->loop->data);
```

Comment:
should this be removed instead of commented out?  looks like a temporary testing change that got committed.